app/uk/gov/hmrc/helptosavestub/controllers/SquidController.scala

```diff
@@ -175,10 +176,12 @@ class SquidController @Inject()(val messagesApi: MessagesApi) extends BaseContro
   }
 
   def createAccount(): Action[AnyContent] = Action { request =>
-    val mimeType = request.headers.toSimpleMap.get(CONTENT_TYPE)
+    val headerMap = request.headers.toSimpleMap
+    val mimeType = headerMap.get(CONTENT_TYPE)
```

Comment:
Yes. They are used to check that the mime type and authorization code is present in the header.